src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -783,6 +789,12 @@
             "required": true,
             "type": "string"
           },
+          {
+            "name": "normalize",
+            "in": "query",
+            "required": false,
+            "type": "boolean"
+          },
```

Comment:
nitpick: I think we're not in the habit of this generally, but can we include the default value here?